packages/nimble-components/src/rich-text/editor/index.ts

```diff
@@ -153,9 +146,8 @@ export class RichTextEditor extends FoundationElement implements ErrorPattern {
     private resizeObserver?: ResizeObserver;
     private updateScrollbarWidthQueued = false;
 
-    private readonly markdownParser = this.initializeMarkdownParser();
-    private readonly markdownSerializer = this.initializeMarkdownSerializer();
-    private readonly domSerializer = DOMSerializer.fromSchema(schema);
+    private readonly markdownParser = new RichTextMarkdownParser();
+    private readonly markdownSerializer = new RichTextMarkdownSerializer();
```

Comment:
Can these be static and shared across all component instances? If so that might be a change to make in a follow-up PR. This PR is probably already too large a scope changing folder hierarchy and refactoring implementation at the same-time so lets not grow it further.

Reply:
Yes, It can be static and we will address it in the follow-up PR.